arangod/V8Server/v8-collection.cpp

```diff
@@ -2129,11 +2098,12 @@ static void JS_TruncateVocbaseCol(v8::FunctionCallbackInfo<v8::Value> const& arg
   TRI_V8_TRY_CATCH_BEGIN(isolate);
   v8::HandleScope scope(isolate);
 
-  OperationOptions opOptions;
-
-  opOptions.waitForSync = ExtractBooleanArgument(isolate, args, 1, false);
-  opOptions.truncateCompact = ExtractBooleanArgument(isolate, args, 2, true);
-
+  OperationOptions options;
+  if (args[0]->IsObject()) {
```

Comment:
This does not check that there is at least one argument in `args`, so it may crash here.